### DISCUSSION TITLE
I think it's better if expo router supports route alias

### DISCUSSION THREAD

**Discussion by likeSo** [expo/router#361] [Ideas]
Consider this scene:
<img width="541" alt="image" src="https://user-images.githubusercontent.com/12247340/223906618-b8d8b1f3-3982-4db0-b5b2-1a57c68a274d.png">
Can you tell what each page is?

**Comment by EvanBacon** [comment]
Not sure what this means. If you mean tsconfig paths aliases then we have it in v2.

**Reply by likeSo** [reply]
Absolutely not. I just want to quickly figure out which route file I'm referring to from a bunch of `[id].tsx` and `index.tsx` and `_layout.tsx` files. I mean, is it possible to add an alias to these files? Like `[id](ProductDetail).tsx` and `[id](OrderDetail).tsx`?
